postory/android/Postory/app/src/main/AndroidManifest.xml

```diff
@@ -43,6 +43,7 @@
         <activity
             android:name=".activities.CreatePostActivity"
             android:exported="true" />
+        <activity android:name=".activities.ListFilteredPostsActivity" />
 
         <activity android:name=".activities.ExploreActivity">
 
```

Comment:
Could you add ```android:windowSoftInputMode="adjustPan" ```  to ExploreActivity in Manifest so that keyboard does not cover the layout.